README.md

```diff
@@ -1,9 +1,31 @@
-# LightweightMatrixC\# 
+# LightweightMatrixC\#
 
 ## Preface
-Lightweight fast matrix class in C# based on [http://blog.ivank.net/](http://blog.ivank.net/lightweight-matrix-class-in-c-strassen-algorithm-lu-decomposition.html).  
+Lightweight fast matrix class in C# based on [http://blog.ivank.net/](http://blog.ivank.net/lightweight-matrix-class-in-c-strassen-algorithm-lu-decomposition.html).
 I also did some changes based on the comments on the original page.
 
+## Quickstart
+
+1. Clone the code from GitHub.
+
+```sh
+git clone https://github.com/Bonifatius94/LightweightMatrixCSharp
+cd LightweightMatrixCSharp
+```
+
+2. Compile the code and run unit tests against it.
```

Comment:
Always use `1.` to reduce git diff when re-ordering. It is rendered correctly anyways.